ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v1/fileinput/clientlibs/site/js/fileinputwidget.js

```diff
@@ -104,7 +104,7 @@ if (typeof window.FileInputWidget === 'undefined') {
                     if(elem && isDuplicatePresent){
                         // now check if duplicate present and get its correct index
                         // today all files are wrapped under .guide-fu-fileItem node
-                        index = elem.closest(self.#fileItemSelector).index();
+                        index = $(elem).parent().index()
```

Comment:
Please add test cases.

Reply:
Updated with latest dev code and added test cases.

Reply:
Added test cases